packages/element/src/index.js

```diff
@@ -1,5 +1,6 @@
 export * from './react';
 export * from './react-platform';
 export * from './utils';
+export { default as Platform } from './platform'; 
```

Comment:
ESLint is complaining about this one

```suggestion
export { default as Platform } from './platform';
```